src/commands/android/dotcommands.ts

```diff
@@ -0,0 +1,119 @@
+import colors from 'ansi-colors';
+import {spawn} from 'child_process';
+import * as dotenv from 'dotenv';
+import path from 'path';
+
+import {ANDROID_DOTCOMMANDS} from '../../constants';
+import Logger from '../../logger';
+import {getPlatformName} from '../../utils';
+import {Platform, SdkBinary} from './interfaces';
+import {checkJavaInstallation, getBinaryLocation, getBinaryNameForOS, getSdkRootFromEnv} from './utils/common';
+
+export class AndroidDotCommand {
+  dotcmd: string;
+  args: string[];
+  sdkRoot: string;
+  rootDir: string;
+  platform: Platform;
+  androidHomeInGlobalEnv: boolean;
+
+  constructor(dotcmd: string, argv: string[], rootDir = process.cwd()) {
+    this.dotcmd = dotcmd;
+    this.args = argv.slice(1);
+    this.sdkRoot = '';
+    this.rootDir = rootDir;
+    this.platform = getPlatformName();
+    this.androidHomeInGlobalEnv = false;
+  }
+
+  async run(): Promise<boolean> {
+    if (!ANDROID_DOTCOMMANDS.includes(this.dotcmd)) {
+      Logger.log(colors.red(`Unknown dot command passed: ${this.dotcmd}\n`));
+
+      Logger.log('Run Android SDK command line tools using the following command:');
+      Logger.log(colors.cyan('npx @nightwatch/mobile-helper <DOTCMD> [options|args]\n'));
+
+      Logger.log(`Available Dot Commands: ${colors.magenta(ANDROID_DOTCOMMANDS.join(', '))}\n`);
```

Comment:
We should also add an "Example command" here:

`Example command: npx @nightwatch/mobile-helper android.emulator -avd @nightwatch-android-11`